apiml-sample-extension-package/src/main/resources/manifest.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: apiml-sample-extension
+# Component identifier. This identifier matches artifact path in Zowe Artifactory https://zowe.jfrog.io/.
+id: org.zowe.apiml.sdk.apiml-sample-extension-package
+# Component version is defined in gradle.properties for Gradle project
+# Human readable component name
+title: A sample extension for API ML
+# Human readable component description
+description: JAR that contains a simple controller.
+license: EPL-2.0
+repository:
+  type: git
+  url: https://github.com/zowe/api-layer.git
+build:
+  branch: "{{build.branch}}"
+  number: "{{build.number}}"
+  commitHash: "{{build.commitHash}}"
+  timestamp: "{{build.timestamp}}"
+# The following block contains all the extensions directory path
+# (or file path) that will be included in the API ML
+gatewaySharedLibs:
+    - ../../../../apiml-sample-extension/build/libs/
```

Comment:
Where does all these dots come from?

Reply:
the path points to the `apiml-sample-extension.jar` location, and it is relative path to the root directory where manifest is located